enzy_htp/analysis/residue_pka.py

```diff
@@ -0,0 +1,43 @@
+from typing import List
```

Comment:
Please add the docstring of this module. You can reference the one in stability.py